packages/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -30,10 +32,22 @@ export function AppHeaderDropdown({
     buttonChildren,
     dropdownArrow = (isOpen) => (isOpen ? <DefaultArrowUp /> : <DefaultArrowDown />),
     popoverProps,
+    open,
+    onOpenChange,
     classes: passedClasses,
     ...restProps
 }: Props & StyledComponentProps<CometAdminAppHeaderDropdownClassKeys>): React.ReactElement {
-    const [showContent, setShowContent] = React.useState<boolean>(false);
+    const defaultShowContentState = React.useState<boolean>(false);
```

Comment:
`defaultOpenState`, since you changed the term to `open` elsewhere too.

Reply:
Ah yes, I must have missed that. I removed it.